Frontend/src/pages/home/index.jsx

```diff
@@ -1,20 +1,25 @@
 import { UiIcon } from '@thewca/wca-components'
 import { marked } from 'marked'
 import moment from 'moment'
-import React, { useContext } from 'react'
+import React, { useContext, useState } from 'react'
 import { Button } from 'semantic-ui-react'
 import { CompetitionContext } from '../../api/helper/context/competition_context'
-import styles from './home.module.scss'
+import RegistrationRequirements from '../register/components/RegistrationRequirements'
+import styles from './index.module.scss'
 
 export default function HomePage() {
   const { competitionInfo } = useContext(CompetitionContext)
+  const { showAllRequirements, setShowAllRequirements } = useState(false)
```

Comment:
`useState` returns an array, not an object.
I always think `showAllRequirements` is a function when I see it away from the definition, so I'd recommend `[areAllRequirementsShown, setAreAllRequirementsShown]` or something (but feel free to ignore this).

Reply:
oops, fixed